tests/TestCase/Model/Filter/ValueTest.php

```diff
@@ -46,6 +46,25 @@ public function testSkipProcess()
     /**
      * @return void
      */
+    public function testProcessWithoutField()
+    {
+        $articles = TableRegistry::get('Articles');
+        $manager = new Manager($articles);
+        $filter = new Value('title', $manager, [
+            'field' => []
```

Comment:
We just bumped the major version to 3.0.0. I think it's fine to add the exceptions for cases where it's surely dev error and have them included of 3.0.1.